misspell.py

```diff
@@ -0,0 +1,49 @@
+import re, random
+class Misspell:
+	#Give list of words to mispell
+	def __init__(self, wordList):

```

Comment:
pep8: use lowercase words separated by underscore for variable names.

Reply:
okay , changes applied .